stream-chat-android-client/src/main/java/io/getstream/chat/android/client/models/UploadedImage.kt

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) 2014-2022 Stream.io Inc. All rights reserved.
+ *
+ * Licensed under the Stream License;
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    https://github.com/GetStream/stream-chat-android/blob/main/LICENSE
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.getstream.chat.android.client.models
+
+/**
+ * Represents a successfully uploaded image.
+ *
+ * @param file The URL of the uploaded image.
+ */
+public data class UploadedImage(
+    val file: String,
```

Comment:
We already have more available fields in the open API spec, such as `thumb_url`, `upload_sizes` and `duration`. However, when debugging network traffic I never got back anything other than `duration` which we don't need and `file`, so I omitted the other ones thinking they might only confuse the users.

One situation where it might be possibly useful for them is if they use a custom CDN so feel free to voice your opinion on this.